test/Main.purs

```diff
@@ -67,8 +67,8 @@ i_2 = ilens (\(Tuple _ b) -> Tuple 0 b) (\(Tuple a _) b -> Tuple a b)
 aGrateExample :: forall a b. Grate (Tuple a a) (Tuple b b) a b
 aGrateExample = grate \f -> Tuple (f fst) (f snd)
 
-collectOfTest :: forall f a b. Distributive f => (a -> f b) -> Tuple a a -> f (Tuple b b)
-collectOfTest = collectOf aGrateExample
+collectOfTest :: forall f g a b. Functor f => Distributive g => (a -> g b) -> f a -> g (f b)
+collectOfTest = collectOf cotraversed
```

Comment:
@paf31 hmm, no, it have different type, if we apply ```collectOf``` to ```aGrateExample, it will this type instead:

```
collectOfTest' :: forall f a b. Functor f => (a -> Tuple b b) -> f a -> Tuple (f b) (f b)
collectOfTest' = collectOf aGrateExample
```